docs/detections/session-view.asciidoc

```diff
@@ -0,0 +1,77 @@
+[[session-view]]
+== Session view
+This page describes how to enable and use Session View. Session View is an investigation tool that allows you to view and analyze Linux process data organized
+in a tree according to the Linux logical event model, with processes organized by parentage and time of execution.
+It displays events in a highly readable format inspired by the terminal, making it a powerful tool for monitoring
+and investigating remote session activity on your Linux infrastructure, and understanding user and service behaviour.
+It is currently in beta.
+
+[float]
+[[session-view-data]]
+====== Session View displays:
+1. *Interactive and non-interactive processes:* processes and services with or without a controlling tty.
+2. *User information:* the user that executed each session or process, and any exec user changes.
+3. *User Input:* commands entered by a user into the terminal.
```

Comment:
I'd add "processes resulting from " as a prefix to "commands entered ...".    We don't grab the commands as typed by the user yet (no fingers in bash, no terminal echo/output yet). Instead we see the resulting processes via BPF with strong indicators that they were user-entered (session has a controlling TTY & parent process is reading from the TTY, and resulting process is a process group leader or has a process group leader different from that of its parent).

Reply:
Mike is correct here. "User Input" is inaccurate for this release. I would probably remove this line. Items one and two in the list have us covered.

Reply:
Thanks gents, appreciate the explanation Mike. Line removed as Norrie suggested.